src/Main.java

```diff
@@ -0,0 +1,363 @@
+            if (givenArray[i] > maxElement) {
+                maxElement = givenArray[i];
+            }
+        }
+        return maxElement;
+    }
+
+    public static int differenceBetweenMinAndMaxElementsInArray (int [] givenArray) {
+        return maxElementInArray(givenArray) - minElementInArray(givenArray);
+    }
+
+    public static boolean comparisonElementsTwoArrays(int[] givenArray1, int[] givenArray2) {
+        if (givenArray1.length != givenArray2.length) {
+            return false;
+        }
+        else {
+            for (int i = 0; i < givenArray1.length; i++) {
+                if (givenArray1[i] != givenArray2[i]) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    public static int searchMissedNumberFromZeroToNineInArray (int[] givenArray) {
+        final int[] arrayWithNumbersFromZeroToNine = new int[]{0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+        int missedNumber = -1;
+
+        for (int i = 0; i < arrayWithNumbersFromZeroToNine.length; i++) {
+            int counter = 0;
+            for (int j = 0; j < givenArray.length; j++) {
+                if (arrayWithNumbersFromZeroToNine[i] != givenArray[j]) {
+                     counter++;
+                     if (counter == givenArray.length) {
+                         missedNumber = arrayWithNumbersFromZeroToNine[i];
+                     }
+                }
+                else {
+                    break;
+                }
+            }
+        }
+        return missedNumber;
+
+    }
+
+    public static int[][] generateTwoDimensionalArray(int length, int width) {
+        int[][] array = new int[length][width];
+        //int number = 1;
+        for (int i = 0; i < array.length; i++) {
+            for (int j = 0; j < array[i].length; j++) {
+                array[i][j] = (int) (Math.random() * 10 - 5);
+                /*array[i][j] = number;
+                number ++;*/
+            }
+        }
+        return array;
+    }
+
+    public static void printTwoDimensionalArray(int[][] givenArray) {
+        for (int i = 0; i < givenArray.length; i++) {
+            System.out.println(Arrays.toString(givenArray[i]));
+        }
+        /*System.out.println();
+        for (int i = 0; i < givenArray.length; i++) {
+            for (int j = 0; j < givenArray[i].length; j++) {
+                System.out.print(givenArray[i][j] + "\t");
+            }
+            System.out.println();
+        }*/
+    }
+
+    public static void swapOddAndEvenLinesIn2DArray(int[][] givenArray) {
+        printTwoDimensionalArray(givenArray);
+        for (int i = 0; i < givenArray.length; i++) {
+            int[] tmp = new int[givenArray.length];
+            if (i % 2 != 0) {
+                tmp = givenArray[i-1];
+                givenArray[i-1] = givenArray[i];
+                givenArray[i] = tmp;
+            }
+        }
+        System.out.println();
```

Comment:
why do you need printing here?)
delete it everywhere
you can print something only in main or in method created for printing

Reply:
deleted